collector/docker/Dockerfile

```diff
@@ -1,14 +1,13 @@
 FROM centos:7
 
 WORKDIR /app/
-RUN yum install -y qt5-qtbase-devel
-RUN yum install -y gdb
-COPY gdb_print.sh /app/
```

Comment:
these are used to load kindling plugin and to export information when unexpected exit happens https://github.com/KindlingProject/kindling/pull/398